CHANGES.md

```diff
@@ -4,9 +4,14 @@
 2.3b2 (unreleased)
 ------------------
 
-- NetLoc objects are now comparable. (#88)
+- NetLoc objects are now comparable.
```

Comment:
The changelog seems confused :)

Reply:
Here some links were added in the next lines.

Reply:
Having links to the tickets in the change log makes it easier to access the tickets. We also added the links here to the other issues of this release.

Reply:
Ah, I misread the diff and you just added the numbers ... fair enough.